test/unit/model/test_objective_creator.py

```diff
@@ -7,14 +7,19 @@
 
 from openstf.enums import MLModelType
 from openstf.model.objective_creator import ObjectiveCreator
-from openstf.model.objective import XGBRegressorObjective
+from openstf.model.objective import XGBRegressorObjective, RegressorObjective
 
 
 class TestObjectiveCreator(BaseTestCase):
-    def test_create_objective(self):
+    def test_create_objective_happy(self):
         Objective = ObjectiveCreator.create_objective(MLModelType.XGB)
         self.assertEqual(Objective, XGBRegressorObjective)
 
+    def test_create_objective_all_model_types(self):
```

Comment:
I would suggest to call this test: `test_create_objective_not_implemented_model_type` or something along that line.

Reply:
good idea